src/Parser/DeathKnight/Unholy/Modules/Features/RpPoolingDA.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+
+class RpPoolingDA extends Analyzer {
```

Comment:
Am I right in concluding that this is an analyzer only for Dark Arbiter and there are no other modules for the same talent? If so how about naming it DarkArbiter instead, I think that would make it easier to identify later.

Reply:
There's already another module named DarkArbiter.  It tracks how much RP is dumped while DA is ticking.  That one should probably get renamed at some point to make it easy to see at a glance what's different about it